src/instructlab/utils.py

```diff
@@ -354,7 +361,9 @@ def _load_schema(path: "importlib.resources.abc.Traversable") -> "referencing.Re
             contents=contents, default_specification=DRAFT202012
         )
     except Exception as e:
-        raise NoSuchResource(ref=str(path)) from e
+        # mypy doesn't understand attrs classes fields, see:
+        # https://github.com/python/mypy/issues/5406
+        raise NoSuchResource(ref=str(path)) from e  # type: ignore[call-arg]
```

Comment:
```suggestion
        raise NoSuchResource(str(path)) from e
```

This will make mypy happy.